api/python/provisioner/commands/confstore_export.py

```diff
@@ -54,6 +55,24 @@ class ConfStoreExport(CommandParserFillerMixin):
     input_type: Type[inputs.NoParams] = inputs.NoParams
     description = "Export pillar template data to ConfStore"
 
+    def _encrypt_value(self, key, value):
+        from cortx.utils.security import cipher
```

Comment:
can we use dynamic imports here using importlib?

Reply:
@sradhanand-pati 
Make sure to try this in your next PR. While you do that, ensure the code still fails with some error if the package is missing.

Reply:
Yes. i will do it @mazinamdar @ypise